deploy/helm/tests/__snapshot__/metrics-collector-config-map-fargate_test.yaml.snap

```diff
@@ -507,99 +507,56 @@ Metrics config should match snapshot when using default values:
-              - ^kube_deployment_status_replicas_updated$
-              - ^kube_deployment_status_replicas_unavailable$
-              - ^kube_deployment_status_condition$
-              - ^kube_replicaset_owner$
-              - ^kube_replicaset_created$
-              - ^kube_replicaset_spec_replicas$
-              - ^kube_replicaset_status_ready_replicas$
-              - ^kube_replicaset_status_replicas$
-              - ^kube_statefulset_replicas$
-              - ^kube_statefulset_status_replicas_ready$
-              - ^kube_statefulset_status_replicas_current$
-              - ^kube_statefulset_status_replicas_updated$
-              - ^kube_statefulset_created$
-              - ^kube_daemonset_status_current_number_scheduled$
-              - ^kube_daemonset_status_desired_number_scheduled$
-              - ^kube_daemonset_status_updated_number_scheduled$
-              - ^kube_daemonset_status_number_available$
-              - ^kube_daemonset_status_number_misscheduled$
-              - ^kube_daemonset_status_number_ready$
-              - ^kube_daemonset_status_number_unavailable$
-              - ^kube_resourcequota$
-              - ^kube_node_status_allocatable$
-              - ^kube_node_spec_unschedulable$
-              - ^kube_job_info$
-              - ^kube_job_owner$
-              - ^kube_job_created$
-              - ^kube_job_complete$
-              - ^kube_job_failed$
-              - ^kube_job_status_active$
-              - ^kube_job_status_succeeded$
-              - ^kube_job_status_failed$
-              - ^kube_job_status_start_time$
-              - ^kube_job_status_completion_time$
-              - ^kube_job_spec_completions$
-              - ^kube_job_spec_parallelism$
-              - ^kube_persistentvolume_capacity_bytes$
-              - ^kube_persistentvolume_info$
-              - ^kube_persistentvolume_status_phase$
-              - ^kube_persistentvolume_claim_ref$
-              - ^kube_persistentvolume_created$
-              - ^kube_persistentvolumeclaim_info$
-              - ^kube_persistentvolumeclaim_access_mode$
-              - ^kube_persistentvolumeclaim_status_phase$
-              - ^kube_persistentvolumeclaim_resource_requests_storage_bytes$
-              - ^kube_persistentvolumeclaim_created$
-              - ^kube_pod_spec_volumes_persistentvolumeclaims_info$
-              - ^kube_service_info$
-              - ^kube_service_created$
-              - ^kube_service_spec_type$
-              - ^kube_service_spec_external_ip$
-              - ^kube_service_status_load_balancer_ingress$
-              - ^kube_endpoint_address_not_ready$
-              - ^kube_endpoint_address_available$
-              - ^kube_endpoint_info$
-              - ^kube_endpoint_created$
-              - ^kube_endpoint_ports$
-              - ^kube_endpoint_address$
+            metric:
+            - "not(\nname == \"kube_deployment_created\" or\nname == \"kube_deployment_created\"
```

Comment:
It's a bit unfortunate, that using `IsMatch` in the condition causes this unreadable mess in the rendered file. But I don't see a way around it.

Reply:
workaround is to use > instead of | , where everything will be one giant line condition if it's better